app/boards/shields/boardsource5x12/boardsource5x12.keymap

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+ 
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/keys.h>
+#include <dt-bindings/zmk/bt.h>
+
+#define DEFAULT 0
+#define RAISE  1
+#define LOWER  2 
+
+/ {
+    keymap {
+        compatible = "zmk,keymap";
+
+        default_layer {
+            bindings = <
+                &kp GRAV    &kp N1      &kp N2   &kp N3    &kp N4    &kp N5      &kp N6    &kp N7    &kp N8    &kp N9   &kp N0   &kp BSPC
+                &kp TAB     &kp Q       &kp W    &kp E     &kp R     &kp T       &kp Y     &kp U     &kp I     &kp O    &kp P    &kp DEL
+                &kp ESC     &kp A       &kp S    &kp D     &kp F     &kp G       &kp H     &kp J     &kp K     &kp L    &kp SEMI &kp SQT
+                &kp LSHIFT  &kp Z       &kp X    &kp C     &kp V     &kp B       &kp N     &kp M     &kp COMMA &kp DOT  &kp FSLH &kp RET
+                &kp PIPE    &kp RCTRL   &kp LALT &kp LGUI  &kp LOWER &kp SPACE   &kp SPACE &kp RAISE &kp LEFT  &kp DOWN &kp UP   &mo RIGHT
+                >;
+            };        
+            
+        RAISE {
+            bindings = <
+                &kp GRAV    &kp F1      &kp F2   &kp F3    &kp F4    &kp F5      &kp F6    &kp F7    &kp F8    &kp F9       &kp F10         &kp F11
+                &kp ESC     &kp N1      &kp N2   &kp N3    &kp N4    &kp N5      &kp N6    &kp N7    &kp N8    &kp N9       &kp N0          &kp F12
+                &kp ESC     &kp N4      &kp N5   &kp N6    &kp PLUS  &kp G       &kp H     &kp J     &kp K     &kp L        &kp SEMI        &kp SQT
+                &kp RET     &kp N7      &kp N8   &kp N9    &kp MINUS &kp B       &kp N     &kp M     &kp COMMA &kp DOT      &kp FSLH        &kp RET
+                &reset      &bootloader &kp LALT &kp LGUI  &kp LOWER &kp SPACE   &kp SPACE &kp RAISE &kp LEFT  &kp C_VOL_DN &kp C_VOL_UP    &mo C_PLAY_PAUSE
+                >;
+            };
+        LOWER {
+            bindings = <
+                &kp ESC    &kp F1       &kp F2   &kp F3    &kp F4    &kp F5      &kp F6    &kp F7    &kp F8    &kp F9   &kp F10  &kp F11
```

Comment:
We usually aim to have the `&bt` behaviors bound somewhere in the default keymap somewhere too.